### PR TITLE
Bugfix - normalise training

### PR DESCRIPTION
**Pull request by joefitter**
Added decorator to nomalise certificates/exemptions of in flight tasks

### CODE REVIEW

File: lib/decorators/normalise-training.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  const exemptions = normaliseExemptions(c.exemptions || []);
+
+  return [
+    ...certificates,
+    ...exemptions
+  ];
+}
+
+module.exports = c => {
+  if (c.certificates || c.exemptions) {
```
Comment (lennym) [inline lib/decorators/normalise-training.js:50]: I'm pretty sure this is `c.data.certificates` etc.

Comment (lennym) [reply lib/decorators/normalise-training.js:50]: Also ~c~ `task`

Comment (joefitter) [reply lib/decorators/normalise-training.js]: right you are, I haven't actually tested this :)

File: lib/decorators/normalise-training.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  const exemptions = normaliseExemptions(data.exemptions || []);
+
+  return [
+    ...certificates,
+    ...exemptions
+  ];
+}
+
+module.exports = task => {
+  if (task.data.certificates || task.data.exemptions) {
```
Comment (lennym) [inline lib/decorators/normalise-training.js:50]:
```suggestion
  if (task.data && (task.data.certificates || task.data.exemptions)) {
```

**Pull request merged**